### ISSUE TITLE
MeSH Publication Types (aka genre/forms)

### ISSUE THREAD

**Issue by NancyJean** [LD4P/qa_server#205]
MeSH Publication Types (PTs) are genre/form terms that can be identified in MeSH RDF by  "@type" : "http://id.nlm.nih.gov/mesh/vocab#PublicationType",  NLM uses these MeSH PTs in its cataloging rather than LC genre/form terms.  While we can add a MeSH lookup for genre/form in our profiles that searches the entire MeSH vocab, it would be more useful if that MeSH genre/form lookup could be limited to MeSH PTs.  See http://id.nlm.nih.gov/mesh/D020463 for an example MeSH PT

**Comment by sfolsom** [comment]
@elrayle @eichmann sounds like this is a request for a subauth that limits to only genres (http://id.nlm.nih.gov/mesh/vocab#PublicationType)

**Comment by NancyJean** [comment]
Yes, exactly.  Not sure it's in scope, but it's important for institutions that use MeSH genres/publication types (like NLM :-)).  I don't know how else a user would be able to cull the genres/pub types out from the rest of MeSH without this change.

**Comment by sfolsom** [comment]
This will be addressed when we make progress on https://github.com/cul-it/bcl-up_server/issues/18
